setup.py

```diff
@@ -29,7 +29,9 @@
 
 kerberos_require = ["requests_kerberos"]
 
-all_require = [kerberos_require]
+sqlalchemy_require = ["SQLAlchemy==1.3.22"]
```

Comment:
Should we use dependency range `~=` or `>=` instead of specific version like this?